lib/Wikia/src/Service/User/Auth/HeliosCookieHelper.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Wikia\Service\User\Auth;
+
+use Wikia\HTTP\Response;
+use Wikia\Service\Helios\HeliosClient;
+
+class HeliosCookieHelper implements CookieHelper {
+
+	const ACCESS_TOKEN_COOKIE_NAME = 'access_token';
+	// This is set to 6 months,(365/2)*24*60*60 = 15768000
+	const ACCESS_TOKEN_COOKIE_TTL = 15768000;
+	const NO_COOKIE_PREFIX = '';
```

Comment:
`NO_COOKIE_PREFIX` -> `COOKIE_PREFIX` name should reflect intent not the actual, temporary content.